adding_species_manuscript.tex

```diff
@@ -155,28 +155,33 @@ \section*{Abstract}
 but producing simulations that recapitulate even the main features of genomic datasets remains a major obstacle.
 Today, more realistic simulations are possible thanks to large increases in
 the quantity of available data
-and the sophistication of inference and simulation software,
-but implementing these simulations can require substantial time and specialized knowledge.
+and the sophistication of inference and simulation software.
+However, implementing these simulations can require substantial time and specialized knowledge.
 These challenges are especially pronounced for simulating less well-studied species,
 since it is not always clear what level of realism is sufficient
 to confidently answer a given question, or what information is required
 to produce simulations of that desired realism.
-\Stdpopsim is a community-developed tool that seeks to lower this barrier
+\Stdpopsim is a community-developed framework that seeks to lower this barrier
 by making it easy to simulate complex population genetic models using
 up-to-date information.
-The initial version of the \stdpopsim catalog contained information for 
-6 species, most of which are well-characterized model organisms.
-Here, we report on community-driven efforts to expand the catalog 
+The initial version of \stdpopsim, the species catalog contained information for 
```

Comment:
Missing an "In" at the start of this sentence? The current version makes it sound like stdpopsim is just the catalog.

Reply:
Yes, that's a typo. I'll also try to reword that part of the abstract to better convey that the catalog is only one aspect of stdpopsim